src/Resources/Resources/RoleAssignments/GetAzureRoleAssignmentCommand.cs

```diff
@@ -245,6 +245,11 @@ public override void ExecuteCmdlet()
                 IncludeClassicAdministrators = IncludeClassicAdministrators.IsPresent,
             };
 
+            if (options.Scope == null && options.ResourceIdentifier.Subscription == null)
+            {
+                WriteTerminatingError(OdataHelper.ScopeAndSubscriptionBothNull(options.Scope, options.ResourceIdentifier.Subscription));
```

Comment:
Why we judge (options.Scope == null && options.ResourceIdentifier.Subscription == null twice in L248 and OdataHelper.ScopeAndSubscriptionBothNull.